app/services/bulk_upload/lettings/year2023/row_parser.rb

```diff
@@ -436,7 +436,11 @@ def valid?
 
       fields.each do |field|
         unless errors.include?(field)
-          errors.add(field, error.message)
+          if error.attribute == :owning_organisation_id || error.attribute == :managing_organisation_id || error.attribute == :startdate
```

Comment:
Is there a way to make this less special case-y, or is that just doomed?

Reply:
Yes there is! Thanks!